coursera/extractors.py

```diff
@@ -174,4 +188,41 @@ def _parse_on_demand_syllabus(self, page, reverse=False,
         if modules and reverse:
             modules.reverse()
 
+        course_id = dom['id']
+        try:
+            references_poll= self._get_references_poll(course_id)
+        except:
```

Comment:
This is too broad. Please be more precise about what exception you are expecting here. Network error, parsing error?